templates/cluster-template-oci-addons.yaml

```diff
@@ -234,7 +234,7 @@ data:
                 path: /etc/kubernetes
           containers:
```

Comment:
A snapshot-controller image has been added in [oci-csi-controller-driver.yaml](https://github.com/oracle/oci-cloud-controller-manager/releases/download/v1.25.0/oci-csi-controller-driver.yaml), can we add that please?

Reply:
The v1.26 link is https://github.com/oracle/oci-cloud-controller-manager/releases/download/v1.26.0/oci-csi-controller-driver.yaml that I'll be using.